WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -291,20 +286,28 @@ class AztecPostViewController: UIViewController {
         notificationCenter.removeObserver(self, name: .UIKeyboardWillHide, object: nil)
     }
 
-    func refreshNavigationBarButtons() {
-        refreshBlogPicker()
-        resizeBlogPicker()
+    func refreshInterface() {
+        reloadBlogPickerTitle()
+        reloadEditorContents()
+        resizeBlogPickerTitle()
     }
 
-    func refreshBlogPicker() {
+    func reloadEditorContents() {
+        let content = post.content ?? String()
+
+        titleTextField.text = post.postTitle
+        richTextView.setHTML(content)
+    }
+
+    func reloadBlogPickerTitle() {
         let blogName = post.blog.settings?.name ?? post.blog.url ?? String()
         let titleText = NSAttributedString(string: blogName, attributes: Constants.blogPickerAttributes)
 
         blogPickerButton.buttonMode = isSingleSiteMode ? .singleSite : .multipleSite
         blogPickerButton.setAttributedTitle(titleText, for: .normal)
     }
 
-    func resizeBlogPicker() {
+    func resizeBlogPickerTitle() {
```

Comment:
I just realized that if the site itself doesn't have a title, you get something like this. Currently the hybrid editor will show the site url if no title is set.
![simulator screen shot jan 28 2017 9 41 43 am](https://cloud.githubusercontent.com/assets/373903/22397744/19aa7e34-e53e-11e6-8f22-748f4ea5abfe.png)

Hybrid:
![simulator screen shot jan 28 2017 9 42 47 am](https://cloud.githubusercontent.com/assets/373903/22397749/2a0ea480-e53e-11e6-8af4-9e82214dfbd5.png)